apps/frontend/src/components/proposal/ProposalTableOfficer.tsx

```diff
@@ -684,8 +689,45 @@ const ProposalTableOfficer = ({
       }
     });
 
+  const fetchProposalCoreBasicData = async () => {
```

Comment:
Not sure if we really need to memorize because we are calling this function only when the "Select all" button is clicked. It is not executing in every component re-render. Correct me if I am wrong